src/Client.ts

```diff
@@ -81,10 +81,20 @@ export class Client {
     }
 
     if (params) {
-      const sortedParams = Object.entries(params)
+      const params_with_array_expanded = new URLSearchParams()
+      Object.entries(params)
         .sort()
         .filter((option) => option[0] !== null)
-      fullPath.search = new URLSearchParams(sortedParams).toString()
+        .forEach(([key, value]) => {
+          if (Array.isArray(value)) {
+            value.forEach((value) =>
+              params_with_array_expanded.append(key, value.toString())
```

Comment:
The api expect arrays to be provided as repeated entries e.g. ?selected_partial_offers[]=OFFER1&selected_partial_offers[]=OFFER2 and passenger_name[]=attar&passenger_name[]=Star. Without this change the parameters would be send as a single comma separated value (selected_partial_offers[]=OFFER1,OFFER2) which the api rejects.

Reply:
When did we introduce this change? 🤔

Reply:
selected partial offers has been like this since we implemented it. What change are you thinking of specifically?

Reply:
Ah nvm, I just noticed now the if condition 🤦‍♂️